src/responder/kcm/kcmsrv_ccache_key.c

```diff
@@ -0,0 +1,145 @@
+/*
+   SSSD
+
+   Copyright (C) Red Hat, 2020
+
+   This program is free software; you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation; either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "config.h"
+
+#include <stdio.h>
+#include <talloc.h>
+
+#include "util/util.h"
+#include "responder/kcm/kcmsrv_ccache_pvt.h"
+
+/*
+ * The secrets store is a key-value store at heart. We store the UUID
+ * and the name in the key to allow easy lookups be either key
```

Comment:
Sentence feels truncated.

Reply:
This was the original comment. I changed the end to "by either part." which sounds complete to me.

Reply:
Thanks. There is a mistype though - "b**e** either part"

Reply:
Done.